packages/backend/src/models/UserAttributesModel.ts

```diff
@@ -75,7 +75,9 @@ export class UserAttributesModel {
             );
         }
         if (filters.userUuid) {
-            query.where(`users.user_uuid`, filters.userUuid);
+            query
```

Comment:
this is a cool trick, I feel like it encodes a bit of business logic though. I can't think of a better approach